ui/packages/consul-ui/tests/steps/doubles/model.js

```diff
@@ -29,6 +29,10 @@ export default function(scenario, create, set, win = window, doc = document) {
     .given(['the local datacenter is "$value"'], function(value) {
       doc.cookie = `CONSUL_DATACENTER_LOCAL=${value}`;
     })
+    .given(['a "$value" metrics provider'], function(value) {
+      doc.cookie = `CONSUL_METRICS_PROXY_ENABLE=1`;
+      doc.cookie = `CONSUL_METRICS_PROVIDER=${value}`;
+    })
```

Comment:
Nice! Kinda unrelated to this PR, but I just added another one here in another PR that is 'about Consul' not necessarily 'about an application'.

I'm thinking we should possibly pull these ones out of this file. I think we've slowly added these and not come back to organize a little just as we were concentrating on other things. Just something maybe to discuss/sort at some point.

Reply:
Yea, you mentioned this before. We can move things over. Just need to find the right time.